src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header__a {
+  padding: 8px 0 12px;
+  line-height: 0;
+  text-decoration: none;
+}
+
+.nav__a {
+  position: relative;
+  margin: 0 10px;
+  margin-bottom: 3px;
+  padding: 24px 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  line-height: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+li {
+  display: contents;
+  list-style-type: none;
+}
+
+img {

```

Comment:
avoid using tag selectors. use classes instead

Reply:
is not fixed fro header